cluster-autoscaler/FAQ.md

```diff
@@ -544,6 +559,33 @@ For example:
 
 For a complete list of the feature gates and their default values per Kubernetes versions, refer to the [Feature Gates documentation](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/).
 
+### How can I use ProvisioningRequest to run batch workloads
+
+Provisioning Request (abbr. ProvReq) is a new namespaced Custom Resource that aims to allow users to ask CA for capacity for groups of pods. For a detailed explanation of the ProvisioningRequest API, please refer to the [proposal](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/proposals/provisioning-request.md).
```

Comment:
Please split the line into more, this should not influence how the result is rendered.